lace/archive/gadget_archive.py

```diff
@@ -23,7 +23,9 @@ class GadgetArchive(BaseArchive):
         _load_data(self, pick_sim, drop_sim, z_max, nsamples=None)
     """
 
-    def __init__(self, postproc="Cabayol23", kp_Mpc=None):
+    def __init__(
+        self, postproc="Cabayol23", kp_Mpc=None, update_kp=False, verbose=False
```

Comment:
@jchavesmontero - could you explain why you need to pass update_kp? Shouldn't kp_Mpc be enough?

Reply:
Now, I recompute kp_Mpc by first loading the data with the archive, and then saving the relevant quantities. You can find this in the new script. If I want to recompute the results for the fiducial value, 0.7/Mpc, I need to include this flag. Another option is to remove it and assume that we will never want something new in the saved file, or only allow to recompute the results for 0.7 if the code does not find the saved file. 

Long story short, I wanted a way to recompute the results for the fiducial value in case we need it.

Reply:
let's then call this something like `force_recompute_linP_params` or something like this. 

The current labels make you think that if you want to specify kp_Mpc you should set both the new value and update_kp?

Reply:
Ok ok, that makes sense. I will update it

Reply:
Could you give me an example for when you would like to recompute these, though? I guess for testing changes in CAMB or similar? I would have thought that we could tests these with the scripts in the scripts/ folder though, not sure why we need this flag... but as long as it has a very clear name I don't mind much.

Reply:
@andreufont done!

Reply:
> Could you give me an example for when you would like to recompute these, though? I guess for testing changes in CAMB or similar? I would have thought that we could tests these with the scripts in the scripts/ folder though, not sure why we need this flag... but as long as it has a very clear name I don't mind much.

Sorry, I did not reload the page. If you check out the new scripts, you will see that we recompute the files by using this flag. The idea is that in case we want to modify the info in the files, for example, adding extra info about cosmology or something like that, we could do it. For example, it was useful for me when I included the conversion between kms and Mpc, which is needed for the Nyx archive